assets/js/theme/common/product-details.js

```diff
@@ -269,19 +269,27 @@ export default class ProductDetails {
 
     showProductImage(image) {
         if (_.isPlainObject(image)) {
-            const zoomImageUrl = utils.tools.image.getSrc(
+            const zoomImageUrl = utils.tools.image.getSrcset(
```

Comment:
We are relying on a trick in getSrcset where it returns a string rather than an array if only one size is passed in. Perhaps we should have a comment to that effect here.